db/migrations/0001_initial.py

```diff
@@ -1,17 +1,48 @@
-# Generated by Django 4.0.2 on 2022-06-15 12:37
```

Comment:
Please do not modify existing migration. You should create a new one

It is a good practice to include one migration per app per PR in Django projects since it helps to avoid migrations folder overload in the future. Represent all changes done to models in one migration file per app per PR, please